modules/social_features/social_group/social_group.module

```diff
@@ -2127,6 +2128,17 @@ function social_group_get_allowed_visibility_options_per_group_type($group_type_
   return $visibility_options;
 }
 
+/**
+ * Implements hook_query_alter().
+ */
+function social_group_query_alter(AlterableInterface $query) {
```

Comment:
Hi @VBaldych,
Could you please add more explanation why this solution was chosen in the PR description?
I see we rarely use `hook_query_alter`, couldn't we use `hook_views_query_alter` instead?
Also have you checked `social_group_flexible_group_views_query_alter()`? 
Maybe we can fix this issue  there?

Also, want to clarify with @ronaldtebrake if the direction with fixing it in `query_alter` is acceptable?

Reply:
The reason why I'm using **hook_query_alter**:
GROUPBY doesn't work in **hook_views_query_alter**. This hook is a child of **hook_views_query_alter** and seems like something prevent executing of GROUPBY before executing this hook. This is a reason why I'm choosing **hook_query_alter**

I tried the similar solution in **social_group_flexible_group_views_query_alter** and it doesn't work. Logically why - **social_group_flexible_group** is a child of **social_hook** and all children hooks should being executed after their parents.